src/3rdparty/walkontable/src/border.js

```diff
@@ -616,7 +680,9 @@ class Border {
         removeClass(this[borderElement], 'hidden');
       }
 
-      style.backgroundColor = borderStyle.color;
+      if (hasClass(this[borderElement], `htAnimateCustomBorders${toUpperCaseFirst(borderElement)}`)) {
+        removeClass(this[borderElement], `htAnimateCustomBorders${toUpperCaseFirst(borderElement)}`);
+      }
```

Comment:
Could you remove this `if`? It's no longer necessary :)

Reply:
Removed.